examples/templates/document_intelligence_agent_team/README.md

```
@@ -0,0 +1,139 @@
+
+| Node | Role | Client-Facing | Sub-Agents |
+|------|------|:---:|------------|
+| `intake` | Document intake | ✅ | — |
+| `coordinator` | Queen Bee | ✅ | researcher, analyst, strategist |
+| `researcher` | Worker Bee | ❌ | — |
+| `analyst` | Worker Bee | ❌ | — |
+| `strategist` | Worker Bee | ❌ | — |
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)